README.md

```diff
@@ -44,3 +54,33 @@ By default, stats are logged every 5 seconds, you can adjust that using
 ``STATSD_LOG_EVERY`` which is the number of seconds between logging operations:
 
     STATSD_LOG_EVERY = 5
+
+Set the desired prefix:
+
+    STATSD_PREFIX = 'scrapy'
+
+You can specify which prefixes you want logged if you don't want all scrapy
+stats. The default is an empty list which indicates that all stats should be
+logged. You can, for example, log only downloader and robotstxt exception stats
+by setting ``STATSD_LOG_ONLY`` to ``['downloader', 'robotstxt.exception_count']``.
+
+    STATSD_LOG_ONLY = []
+
+You can also specify prefixes to ignore the same way using ``STATSD_IGNORE``:
+
+    STATSD_IGNORE = []
+
+
+## Tags
+
+Certain platforms such as datadog and influxdb offer tagging options.
+
+To enable tagging set ``STATSD_TAGGING`` to ``True``, it is disabled by default:
+
+    STATSD_TAGGING = False
+
+Then, you can set custom tags using ``STATSD_TAGS``:
+
+    STATSD_TAGS = {
+        'spider_name': True
```

Comment:
It's not clear to me from this whether the tag will be `spider_name` (and you're supposed to replace that with the actual spider name) or if `spider_name` is a keyword that automatically inserts the spider name.  If the latter, are there any other keywords?  Also, what does `True` mean here?